Documentation/operations/upgrade.rst

```diff
@@ -300,14 +300,13 @@ Removed Options
 * The previously deprecated high-scale mode for ipcache has been removed.
 * The previously deprecated hubble-relay flag ``--dial-timeout`` has been removed.
 * The previously deprecated External Workflows feature has been removed.
+* The previously deprecated ``--datapath-mode=lb-only`` for plain Docker mode has been removed.
```

Comment:
So the flag was actually deprecated in 1.17? From the removed note it sounds like it wasn't deprecated in a released version yet.

Reply:
Yes, it got deprecated in 1.17.0 via https://github.com/cilium/cilium/pull/37391 .